docs/src/components/social-links-type.astro

```diff
@@ -0,0 +1,7 @@
+---
+import { socialLinks } from '../../../packages/starlight/schemas/social';
+
+const socials = [...socialLinks].sort((a, b) => a.localeCompare(b, 'en'));
```

Comment:
Relying on `localeCompare()` in the compare function instead of a basic `sort()` to ensure a case-insensitive sort so that `codeberg` appears before `codePen` in the list for example.

Reply:
Looks good. Initially, I wondered if we should sort with the current page’s locale instead of always `en`, but on second thoughts, given this list’s contents, this makes sense 👍